<launch>
  <!-- Run multiple cameras -->
  <include file="$(find camera_tracking)/launch/cv_camera.launch" />

  <node pkg="camera_tracking" type="tf_parser" name="tf_parser" output="screen">
    <rosparam param="camera_names">[testcam]</rosparam>
    <rosparam param="tag_names">[tag_0, tag_1, tag_69]</rosparam>
  </node>
</launch>
